packages/create-cloudflare/src/frameworks/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": "../../tsconfig.json"
```

Comment:
This file is needed so that VS Code (eslint) knows that the TS files in this folder are configured by TypeScript. The package.json in this directory makes eslint think this is a separate TS project.